extensions/local-switch/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "local-switch",
+  "title": "Local Switch",
+  "description": "Quickly switch between localhost and remote urls, preserving url path",
+  "icon": "icon.png",
+  "author": "ewgenius",
+  "categories": [
+    "Developer Tools",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "switch",
+      "title": "Switch Host",
+      "subtitle": "Local Switch",
```

Comment:
you'll probably want to remove the subtitle if we rename the extension to `Host Switch`, to avoid needless repetition 🤔

Reply:
Sure, removed. Also renamed command to match extension name